### ISSUE TITLE
A11y_ Windows Template Studio_ Navigation pane details_AI4D: Name property assigned to link (Blank optional) is not correct.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/WindowsTemplateStudio#3899]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#GH_WindowsTemplateStudio_Oct2020;#Windows Template Studio;#A11y_Windows Template Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#AI4D;#NonBenchmark;#AILimited;

### **Environment Details:**
Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.8.0 Preview 3.1
Windows Version: Windows10

### **Repro Steps:**

1. Open Visual Studio 2019 
2. Navigate to File from menu with the help of tab key and press enter key to expand it. 
3. Select new, press enter to expand and then select project.
4. Select the "Windows template studio (WPF.Net Core)". Navigate to next button with the help of tab key and press enter key
5. "Configure your new project" screen will appear. 
6. Navigate to create button with the help of tab key and press enter key.
7. New WPF App screen will open.
8. Navigate to Navigation pane with the help of tab key
9. Using Tab key navigate to details and press enter
10. Open AI4D AND navigate to “blank optional “with the help of Tab key and check its name property

### **Actual Result:**
Name property to hyperlink which is after layout is assigned as “space character” and because of it when screenreader user navigates to blank optional with the help of tab key, screenreader announces “ hyperlink space”

### **Note:**
This issue is observed in following scenarios:
•	Blank details
•	Menu Bar Details
•	Ribbon Details

### **Expected Result:**
The name property must not only contain space character. Relevant name property should be assigned like “Blank optional” to the link which is next to layout.

### **User Impact:**
Screenreader user will not get proper information about the link as name property only contains space character.

**Comment by sibille** [comment]
Verified in dev-nightly version:
Wizard version: 0.22.20310.1
You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by Yash14j** [comment]
Issue does not repro in the Wizard version: 0.22.20310.1 hence closing this bug.
